docs-2.0/3.ngql-guide/16.subgraph-and-path/2.find-path.md

```diff
@@ -2,6 +2,10 @@
 
 The `FIND PATH` statement finds the paths between the selected source vertices and destination vertices.
 
+!!! note
+
+    To improve the query performance with the `FIND PATH` statement, you can add the `num_operator_threads` option in the `nebula-graphd.conf` configuration file. The value range of the `num_operator_threads` option is [2, 10] and make sure that the value is not greater than the number of CPU cores of the machine where the `graphd` service is deployed. It is recommended to set the value to the number of CPU cores of the machine where the `graphd` service is deployed. For more information about the `nebula-graphd.conf` configuration file, see [nebula-graphd.conf](../../5.configurations-and-logs/1.configurations/3.graph-config.md).
```

Comment:
They are called parameters in the configuration docs, not options. Please keep consistency.